api/OfflineAudioContext.json

```diff
@@ -52,14 +96,26 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/OfflineAudioContext/OfflineAudioContext",
           "description": "<code>OfflineAudioContext()</code> constructor",
           "support": {
-            "chrome": {
-              "version_added": "55",
-              "notes": "Before Chrome 59, the default values were not supported."
```

Comment:
I couldn't figure out what these notes might be referring to so I removed them. Expecting to be asked by a reviewer to dig deeper, but taking my chances :)

Reply:
I gave a quick look for something which might even hint as to what this could be about and found nothing. I think this is the right call. 👍